docs/repositories-configure/configuring-code-patterns.md

```diff
@@ -1,57 +1,48 @@
+description: Configure the static analysis tools and code patterns that Codacy uses to analyze your repositories to ensure that the analysis results fit your needs.
 ---
 
 # Configuring code patterns
 
-By default, Codacy uses a subset of the supported static analysis tools and code patterns to analyze your repositories. These default settings result from community feedback or existing coding standards. However, you can adapt the default settings to your scenario by configuring the tools and code patterns that Codacy uses to analyze your repository.
+{% include-markdown "../assets/includes/admin-access-control-info.md" %}
 
-!!! note
-    -   Organization admins can [change who is allowed to configure code patterns](../organizations/roles-and-permissions-for-organizations.md#change-analysis-configuration).
+By default, Codacy analyzes your repositories using a subset of the supported analysis tools and code patterns. These defaults are based on current best practices and community feedback, and you can adapt them to your needs in several ways:
+
+-   [Configuring tools and code patterns using the Codacy UI](#configuring-tools-and-code-patterns-using-the-codacy-ui)
+-   [Importing configurations from another repository](#import-patterns)
+-   [Using tool configuration files](#using-your-own-tool-configuration-files)
 
-    -   If your repository is following an [organization coding standard](../organizations/using-coding-standards.md), changes made to any tool or code pattern cause the repository to stop following the coding standard. In this case Codacy asks for your confirmation before accepting the changes, and then copies the coding standard configurations to your repository so you can customize them.
+## Configuring tools and code patterns using the Codacy UI
 
-To configure the tools and code patterns for your repository:
+!!! note
+    If you update the configurations of a repository that follows a [coding standard](../organizations/using-coding-standards.md), Codacy copies the coding standard configurations to the repository and the repository stops following the coding standard. You can then customize the repository configurations without affecting the coding standard.
+
+To configure the tools and code patterns for a repository using the Codacy UI:
 
 1.  Open your repository **Code patterns** page.
 
     ![Code patterns page](images/code-patterns.png)
 
-1.  Toggle the tools that Codacy will use to analyze your repository.
+1.  Enable or disable the tools that Codacy should use to analyze the repository.
 
     ![Toggling tools](images/code-patterns-toggle-tools.png)
 
-1.  Select each tool to configure and toggle the corresponding code patterns using the checkbox next to each pattern.
-
-    !!! tip
-        Codacy displays the tag **New** for one month next to the name of any recently added code patterns.
+1.  Select a tool to enable or disable its code patterns.
```

Comment:
The suggestion below tries to:

- Avoid having two paragraphs below the step (implies removing line 32)
- Keep the value of using the filter (to make it easier to find relevant patterns)

However, we'll get a bigger text block for the step... Do evaluate if it makes sense to you. 🙏 

```suggestion
1.  Select a tool to enable or disable its code patterns. To make it easier to find relevant patterns, use the sidebar filters. You can filter by language, issue category, or status.
```

Reply:
Also, we could link "issue category" to the [categories description](https://docs.codacy.com/faq/code-analysis/which-metrics-does-codacy-calculate/#issues), as we dropped that part from this page.